tests/utils.py

```
@@ -34,7 +30,6 @@
 _SAMPLE_VOCAB_DIR = _SAMPLE_DATA_DIR / "distilbert-tokenizer-files"
 _CHECKPOINT_PATH = str(_SAMPLE_VOCAB_DIR)
 SIMPLE_PERTURBATION_TESTING_CONFIG = BehavioralTestingOptions(
-    neutral_suffix=NeutralTokenOptions(suffix_list=["pls", "thanks"], prefix_list=["pls", "hello"]),
```

This had no effect because of a typo. It should have been `neutral_token` instead of `neutral_suffix`. Since I changed `BehavioralTestingOptions`'s parent class from `BaseModel` to `BaseSettings`, in which extra arguments are forbidden by default instead of ignored, this was now crashing.
Alternatively, I could fix the typo to `neutral_token` and adapt the two tests that depend on it. Any preference?

---

Resolved in 7db40b04aab5e53ba11aafb0db2fa0b446258992